src/UpdateCounterMigrationBundle/Command/GenerateMigrationConfigCommand.php

```diff
@@ -55,13 +57,15 @@ protected function execute(InputInterface $input, OutputInterface $output)
         if (0 === count($migrationConfigData)) {
             $output->writeln('No bundles need to be migrated. Off to the pub!');
 
-            return;
+            return 0;
         }
 
         $output->writeln('chameleon_system_update_counter_migration:');
         $output->writeln('  mapping:');
         foreach ($migrationConfigData as $source => $target) {
             $output->writeln('    - '.$source.': '.$target);
         }
+
+        return 0;
```

Comment:
Symfony commands should return exit-codes. Returning 0 makes the return value valid while not changing the behaviour (no return is handled as 0 for backwards compatibility reasons)